File: Beginner/1179 - Array Fill IV/1179.py
par = []
impar = []

for _ in range(15):
    numero = int(input())
    
    if numero % 2 == 0:
        par.append(numero)
        if len(par) == 5:
            for i in range(5):
                print(f"par[{i}] = {par[i]}")
            par = []
    else:
        impar.append(numero)
        if len(impar) == 5:
            for i in range(5):
                print(f"impar[{i}] = {impar[i]}")
            impar = []

for i in range(len(impar)):
    print(f"impar[{i}] = {impar[i]}")

for i in range(len(par)):
    print(f"par[{i}] = {par[i]}")

File: Beginner/1179 - Array Fill IV/1179.c
#include <stdio.h>

int main()
{

    int i, i2, N, odd = 0, even = 0;
    int oddV[5], EvenV[5];

    for (i = 0; i < 15; i++)
    {
        scanf("%i", &N);
        if (N % 2 == 0)
        {
            EvenV[even] = N;
            even++;
            if (even == 5)
            {
                for (i2 = 0; i2 < even; i2++)
                {
                    printf("par[%i] = %i\n", i2, EvenV[i2]);
                }
                even = 0;
            }
        }
        else
        {
            oddV[odd] = N;
            odd++;
            if (odd == 5)
            {
                for (i2 = 0; i2 < odd; i2++)
                {
                    printf("impar[%i] = %i\n", i2, oddV[i2]);
                }
                odd = 0;
            }
        }
    }
    for (i2 = 0; i2 < odd; i2++)
    {
        printf("impar[%i] = %i\n", i2, oddV[i2]);
    }
    for (i2 = 0; i2 < even; i2++)
    {
        printf("par[%i] = %i\n", i2, EvenV[i2]);
    }
    return 0;
}